### PR TITLE
[W21] Ray Lin: 05/23 ~ 05/27

### PR DESCRIPTION
**Pull request by raylin**

### CODE REVIEW

File: RayLin.md [hunk 1]
```diff
@@ -1,22 +1,16 @@
...
-    - Help investigate and find regression
+    - ask visual for feedback
   - Bug 1091241 - Make add-on detail page look more like a detail page
-    - r? Margaret
-    - fixed border issue, review again
+    - FIXED
   - Bug 1267935 - Video controls are not correctly scaled for several video types
-    - approval-mozilla-aurora?
+    - uplifted to Aurora
+    - video control is scaled fine in latest Aurora build(05/29) now
```
Comment (timdream) [inline RayLin.md]: Nice! Thanks :)

File: RayLin.md [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
-    - WIP
-    - Three approaches to vertically move up subtitle while control bar is showing:
-      1. addEventListener when frameUpdate - has a pitfall that we could not re-style caption box when video is paused
-      2. expose an attribute like "mozCaptionBox" on \<video\> element to let control adjust caption box.
-    - Currently work on second approach
-  - Bug 1203481 - All dividers in the URL bar should have the same style
-    - r+
+    - r? UI issues are fixed in the main
+    - next week:
+      - ask visual for new CC button icon, since transparency PNG could not be brighter even opacity is set to 1
```
Comment (timdream) [inline RayLin.md]: Do you need me to ask around?

Comment (raylin) [inline RayLin.md]: I just asked(NI) :shorlander at bugzilla but "Yes", that would be good if it can speed up the process. Thanks!

Comment (timdream) [inline RayLin.md]: Emailed Harly.

**Pull request merged**